src/Uno.UI/UI/Xaml/Media/AcrylicBrush/Android/CustomRealtimeBlurView.cs

```diff
@@ -0,0 +1,428 @@
+					{
+						return false;
+					}
+
+					r = true;
+				}
+				catch (Java.Lang.OutOfMemoryError)
+				{
+					// Bitmap.createBitmap() may cause OOM error
+					// Simply ignore and fallback
+				}
+				finally
+				{
+					if (!r)
+					{
+						Release();
+					}
+				}
+				if (!r)
+				{
+					return false;
+				}
+			}
+
+			if (dirty)
+			{
+				if (mBlurImpl.Prepare(Context, mBitmapToBlur, radius))
+				{
+					mDirty = false;
+				}
+				else
+				{
+					return false;
+				}
+			}
+
+			return true;
+		}
+
+		protected void Blur(Bitmap bitmapToBlur, Bitmap blurredBitmap)
+		{
+			mBlurImpl.Blur(bitmapToBlur, blurredBitmap);
+		}
+
+		protected View GetActivityDecorView()
+		{
+			Context ctx = Context;
+			for (int i = 0; i < 4 && ctx != null && !(ctx is Activity) && ctx is ContextWrapper; i++)
+			{
+				ctx = ((ContextWrapper)ctx).BaseContext;
+			}
+			if (ctx is Activity)
+			{
+				return ((Activity)ctx).Window.DecorView;
+			}
+			else
+			{
+				return null;
+			}
+		}
+
+		protected override void OnAttachedToWindow()
+		{
+			base.OnAttachedToWindow();
+			mDecorView = GetActivityDecorView();
+			if (mDecorView != null)
+			{
+				mDecorView.ViewTreeObserver.AddOnPreDrawListener(_preDrawListener);
+				mDifferentRoot = mDecorView.RootView != RootView;
+				if (mDifferentRoot)
+				{
+					mDecorView.PostInvalidate();
+				}
+			}
+			else
+			{
+				mDifferentRoot = false;
+			}
+		}
+
+		//@Override
+		protected override void OnDetachedFromWindow()
+		{
+			if (mDecorView != null)
+			{
+				mDecorView.ViewTreeObserver.RemoveOnPreDrawListener(_preDrawListener);
+			}
+			Release();
+			base.OnDetachedFromWindow();
+		}
+
+		//@Override
+		public override void Draw(Canvas canvas)
+		{
+			if (mIsRendering)
+			{
+				// Quit here, don't draw views above me
+				//throw STOP_EXCEPTION;
+				return;
+			}
+			else if (RENDERING_COUNT > 0)
+			{
+				// Doesn't support blurview overlap on another blurview
+			}
+			else
+			{
+				base.Draw(canvas);
+			}
+		}
+
+		protected override void OnDraw(Canvas canvas)
+		{
+			base.OnDraw(canvas);
+			DrawBlurredBitmap(canvas, mBlurredBitmap, mOverlayColor);
+		}
+
+		///**
+		// * Custom draw the blurred bitmap and color to define your own shape
+		// *
+		// * @param canvas
+		// * @param blurredBitmap
+		// * @param overlayColor
+		// */
+		protected void DrawBlurredBitmap(Canvas canvas, Bitmap blurredBitmap, int overlayColor)
+		{
+			if (blurredBitmap != null)
+			{
+				mRectSrc.Right = blurredBitmap.Width;
+				mRectSrc.Bottom = blurredBitmap.Height;
+				mRectDst.Right = Width;
+				mRectDst.Bottom = Height;
+				canvas.DrawBitmap(blurredBitmap, mRectSrc, mRectDst, null);
+			}
+			mPaint.Color = new Color(overlayColor);
+			canvas.DrawRect(mRectDst, mPaint);
+		}
+
+		public void Destroy()
+		{
+			if (mDecorView != null)
+			{
+				mDecorView.ViewTreeObserver.RemoveOnPreDrawListener(_preDrawListener);
+			}
+
+			Release();
+		}
+
+		private class StopException : Exception
```

Comment:
Codacy found an issue: [Make this exception 'public'.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5932087)